### DISCUSSION TITLE
Phrozen Mega 8k support

### DISCUSSION THREAD

**Discussion by marchermitte** [sn4k3/UVtools#744] [Q&A] [answered]
Hello!
Thank you for this amazing tool, I just found out about UVtool and am starting using It.
I own a sonic 4k, which is in the supported printers list, but also a mega 8k and that one seems NOT to be supported. Is there a workaround or some config to enable this printer in UVtool? It does use CTB V.4 so, can you confirm that It will definitely not be supported?
I was interested to add the exposure time finder in my workflow, It's remarkable! 

Thank you.

**Comment by sn4k3** [comment, marked as answer]
That list is not the whole picture, also it has 3 lines that contain a dozen of printers

![image](https://github.com/sn4k3/UVtools/assets/113281/8080c307-b105-4226-b2a9-f1b4faa9b85d)

Still the recent CTB is found not very compatible with that test, you have to try a dry test to confirm if it works or not

**Comment by marchermitte** [comment]
All hope is not dead yet then... I will try and let you know.
thank you for your answer!

**Comment by marchermitte** [comment]
It works with analyzing and fixing layers, export and print without any problem. I didn't test the multiple exposure yet... Thank you for the tool!
